runtime/queries/cmake/highlights.scm

```diff
@@ -21,7 +21,7 @@
 
 (normal_command (identifier) @function)
 
-["ENV" "CACHE"] @storageclass
+["ENV" "CACHE"] @keyword.storage
```

Comment:
```suggestion
["ENV" "CACHE"] @module
```

Reply:
Env and cache in cmake isn't exactly module if i understand correctly

Reply:
They're containers.